### ISSUE TITLE
Is differentiating BLAS calls supported?

### ISSUE THREAD

**Issue by roastduck** [EnzymeAD/Enzyme#1249]
Just a quick question: is differentiating BLAS calls supported? I found some code related to BLAS, but I didn't find any documentation.

**Comment by tgymnich** [comment]
Yes some level 1 functions like, `dot`, `axpy`, `copy`, `asum`,   and `scal` should be supported. @ZuseZ4  is working on extending our BLAS support https://github.com/EnzymeAD/Enzyme/pull/1208.

See: https://github.com/wsmoses/Enzyme/blob/4f95c0ff247dccde5cf1669be1c8bfa39831b851/enzyme/Enzyme/BlasDerivatives.td
